src/adaptation/stannuts.jl

```diff
@@ -63,21 +63,21 @@ function adapt!(tp::StanNUTSAdaptor, θ::AbstractVector{<:Real}, α::AbstractFlo
 
     # Ref: https://github.com/stan-dev/stan/blob/develop/src/stan/mcmc/hmc/nuts/adapt_diag_e_nuts.hpp
     if is_in_window(tp)
-        adapt!(tp.pc, θ, α, false)
-    elseif is_window_end(tp)
-        # TODO: consider make the boolean variable as part of reset! (similar to what happens to μ)
-        adapt!(tp.pc, θ, α, true)
+        if is_window_end(tp)
+            adapt!(tp.pc, θ, α, true)
```

Comment:
Consider adding a comment here, since the flag's meaning (i.e. `true`/`false`) is not immediately clear.